rnd/market/schema.prisma

```diff
@@ -53,12 +53,11 @@ model AnalyticsTracker {
 }
 
 model FeaturedAgent {
-  id          String  @id @unique @default(dbgenerated("gen_random_uuid()")) @db.Uuid
-  agentId     String  @unique @db.Uuid
-  agent       Agents  @relation(fields: [agentId], references: [id])
-  is_featured Boolean @default(false)
-  category    String  @default("featured")
-
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
+  id                 String   @id @unique @default(dbgenerated("gen_random_uuid()")) @db.Uuid
```

Comment:
This seems to be working only in postgres, have we tried it on sqlite?
Other table uses @default(uuid())

Reply:
Market is supposed to only run on our servers so only supporting Postgres for now. Ideally we’d do it in Supabase at some point

Reply:
degenerated allows you to insert without having prisma open to generate the ids